pyproject.toml

```diff
@@ -32,12 +32,14 @@ dependencies = [
   "dask[array]>=2021.4.1",
   "fsspec>=2022.8.0",
   "imageio[ffmpeg]>=2.11.0,<2.28.0",
+  "ngff-zarr>=0.8.2",
   "numpy>=1.21.0,<2.0.0",
   "ome-types[lxml]>=0.4.0",
   "ome-zarr>=0.6.1",
   "semver>=3.0.1",
   "tifffile>=2021.8.30",
-  "zarr>=2.6.0,<3.0.0",
+  # this pin <2.18.0 should be temporary and is maybe related to https://github.com/zarr-developers/zarr-python/issues/1891
+  "zarr>=2.6.0,<2.18.0",
```

Comment:
sadly, without this version pin, unit tests were failing.  I don't know what the fix is but we should probably have a "reminder" issue to try to unpin it again later.